src/Components/InputBook.js

```diff
@@ -21,7 +21,10 @@ function InputBook() {
     }, 1000);
   };
   return (
-    <div>
+    <div className="input-form">
```

Comment:
**NB: I will leave the comment here because you did not make the file visible in this PR:**:
- **Optional**: Kindly remove the commented code to make your work looks cleaner and professional.

 
![Capture1](https://user-images.githubusercontent.com/63972670/201243561-26aa0e6a-a340-47b8-bef1-0d9660e285b7.PNG)

Reply:
**NB: I will leave the comment here because you did not make the file visible in this PR:**

- **OPTIONAL**: Please at this level, you need to get used to ES class because you are using ES6 Javascript. Otherwise, this is not going to help you later ahead.  You have used this method in most of your work, not good. 
-
![Capture](https://user-images.githubusercontent.com/63972670/201246176-9b5d9bf7-14a7-4900-b9dc-4f0aad955cf9.PNG)

Reply:
All functions changed to ES6 arrow functions